xml/System.IO.Pipelines/PipeReader.xml

```diff
@@ -329,7 +345,7 @@
         <Parameter Name="cancellationToken" Type="System.Threading.CancellationToken" />
       </Parameters>
       <Docs>
-        <param name="cancellationToken">To be added.</param>
+        <param name="cancellationToken">The token to monitor for cancellation requests. The default value is <see langword="null" />.</param>
```

Comment:
CancellationToken is a struct so this should be

```suggestion
        <param name="cancellationToken">The token to monitor for cancellation requests. The default value is <see langword="default" />.</param>
```

or

```suggestion
        <param name="cancellationToken">The token to monitor for cancellation requests. The default value is <see langword="CancellationToken.None" />.</param>
```

Reply:
Oh I see, I didn't check if it was a struct.

Then I don't understand why the default value is specified in line 300 in this same document as `null`:

```xml
<MemberSignature
    Language="C#"
    Value="public abstract ValueTask<ReadResult> ReadAsync (CancellationToken cancellationToken = null);" />
```

@mairaw do you know why this is being set to null? In the code, it is `default` (see [here](https://github.com/dotnet/corefx/blob/e57dd4784a83d3310a14e7c4b274885b91c521fd/src/System.IO.Pipelines/src/System/IO/Pipelines/PipeReader.cs#L27)). Is this a bug?

Reply:
Not sure. I'll open a bug for @joelmartinez to investigate.